src/main/resources/application.properties

```diff
@@ -1 +1,20 @@
 spring.application.name=epic-board-games-shop
+
+
+# Ustawienia bazy danych
+spring.datasource.url=jdbc:mysql://localhost:3306/ebgs_db
+spring.datasource.username=root
+spring.datasource.password=password
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+
+# Tymczasowa konfiguracja Hibernate, pozniej zamienione na liquidBase
+spring.jpa.hibernate.ddl-auto=update
+spring.jpa.show-sql=true
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL8Dialect
+
+
+
+
+
+# docker run --name mysql_ebgs -e MYSQL_ROOT_PASSWORD=password -e MYSQL_DATABASE=ebgs_db -v EBGS-DB-volume:/var/lib/mysql -p 3306:3306 -d mysql
```

Comment:
Przerzućmy gdzieś tę komendę i komentarz (najlepiej readme.md), oraz zmieńmy komentarz na angielski

Reply:
A najlepiej to zróbmy odpalenie tego w docker-compose

Reply:
przeniosłem do osobnego pliku który dałem do gitignora. Docker-compose jeszcze bedzie do zrobienia